nestjs/src/courses/course-students/course-students.service.ts

```diff
@@ -0,0 +1,177 @@
+import { User } from '@entities/user';
+import { MentorBasic, StageInterviewFeedbackJson } from '@common/models';
```

Comment:
Please do not import from `@common/` 
If you need a model, just copy to nest. We want to kill `@common`

Reply:
@valerydluski

Reply:
@Alphajax When you asked me, I thought you meant Entity ([example](https://github.com/rolling-scopes/rsschool-app/blob/master/server/src/models/mentor.ts#L22)). We can confidently move the rest to Nest. I apologize for the misunderstanding, my bad.

Reply:
@valerydluski thank you fo explanation